homeassistant/components/powerwall/entity.py

```diff
@@ -26,7 +25,6 @@ def __init__(self, coordinator, site_info):
         unique_group = (
             site_info[SITE_INFO_UTILITY],
             site_info[SITE_INFO_GRID_CODE],
-            str(site_info[SITE_INFO_NOMINAL_SYSTEM_POWER_KW]),
```

Comment:
We will need a migration. Now all old entries will occupy the entity ID and will show up as unavailable while new sensors are showing up too. 

You can use `helpers.entity_registry.async_migrate_entries(…)` if it's config entry based.

Reply:
This integration hasn't been released yet. Then we don't need a migration, right?

Reply:
Right